jdi-light-bootstrap-tests/pom.xml

```diff
@@ -25,6 +25,11 @@
             <artifactId>jdi-light-bootstrap</artifactId>
             <version>1.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.epam.jdi</groupId>
+            <artifactId>jdi-light-html</artifactId>
+            <version>1.1.0</version>
+        </dependency>
```

Comment:
We don't need this dependency because "jdi-light-html" is already part of "jdi-light-bootstrap" package